packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -335,6 +346,10 @@ function UserMenu({
         )}
         usePortal={false}
       >
+        <li role="none" className={mfaBannerStyle}>
```

Comment:
Should this be conditionally rendered based on whether or not they've already signed up for MFA

Reply:
So this was actually changed from the original design because we realized that we wouldn't be able to propagate that boolean value easily to the rest of the services that rely on UserMenu. Instead, we'll temporarily show the banner for a set amount of time (~30 days or so) and then remove it afterwards.

Reply:
Sounds good, TY for the clarity

Reply:
@kevintian Is there a reason we don't move the logic to the API backing the navigation? It would propagate across other services if we made that change.

Reply:
That's a good point - so my one concern is that while Stitch, Charts, and Cloud use the UserMenu component and call the API backing the navigation (and thus would be updated properly), University uses a fork of UserMenu and would have to update their component each time regardless.

I'm fine with moving the logic to the API backing the navigation to make things easier for Stitch/Charts/Cloud if desired!

Reply:
Deferring to you here @DesignerDave

Reply:
I'd vastly prefer we move this from a prop to the API. University has the option of using the user menu we export as well, but since they don't, how we approach this here is unrelated to their specific case.